pydantic/utils.py

```diff
@@ -83,6 +83,15 @@ def in_ipython() -> bool:
         return True
 
 
+def deep_update(mapping: Dict[str, Any], updating_mapping: Dict[str, Any]) -> Dict[str, Any]:
+    for k, v in updating_mapping.items():
+        if (k in mapping) and isinstance(mapping[k], dict) and isinstance(v, dict):
```

Comment:
```suggestion
        if k in mapping and isinstance(mapping[k], dict) and isinstance(v, dict):
```

I don't think parentheses are required here.